common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/responses/resetpassword/ResetPasswordContinueApiResponse.kt

```diff
@@ -57,11 +56,6 @@ class ResetPasswordContinueApiResponse(
      * @see com.microsoft.identity.common.java.nativeauth.providers.responses.resetpassword.ResetPasswordContinueApiResult
      */
     fun toResult(): ResetPasswordContinueApiResult {
-        LogSession.logMethodCall(
```

Comment:
Since we're doing quite some API response validation in these `toResult()` methods, I'm wondering whether it's useful to log the result of that. Perhaps that logging can be done in the Interactor; a log through which we can determine how the SDK has parsed the API response and what ApiResult was generated accordingly.

Reply:
Put rawResponse -> apiResult logging in interactors.